src/models/user.ts

```diff
@@ -32,23 +32,39 @@ userSchema.methods.checkPass = async function (password:string) {
 const Profile = mongoose.model(
     'Profile',
     new Schema({
-        firstName: {
+        email: {
```

Comment:
Since each user will have their emails which is not editable,

You can safely remove this field since it would mean requiring keeping in sync two fields on two different models that are already linked.